src/main/java/javapoz24/team3/rental/application/config/MockData.java

```diff
@@ -137,6 +145,34 @@ public void GenerateMockData() {
         carService.addOrUpdate(CarDTO.fromCar(car2));
         carService.addOrUpdate(CarDTO.fromCar(car3));
 
+        Customer cust1 = Customer.builder()
+                .id(null)
+                .firstName("Grzegorz")
+                .lastName("Brzęczyszczykiewicz")
+                .email("gb@wp.pl")
+                .address(customerAddress1)
+                .build();
+
+        Customer cust2 = Customer.builder()
+                .id(null)
+                .firstName("Rafał")
+                .lastName("Anielewicz")
+                .email("ra@op.pl")
+                .address(customerAddress2)
+                .build();
+
+        Customer cust3 = Customer.builder()
+                .id(null)
```

Comment:
Jak używasz buildera to nie ma sensu ustawiać id na null.

Reply:
Rozumiem, że będzie ustawiony domyślnie?
To są dane testowe, może za chwilę będę chciał podać jednak id, żeby nastąpił update więc zostawiłbym jak jest.